starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -279,15 +279,15 @@ private GetOutputFormatResult() {
     }
 
     private boolean formatHasCropValues() {
-      if (!mFormatHasCropValues.isPresent()) {
+      if (!mFormatHasCropValues.isPresent() && mFormat != null) {
         boolean hasCropValues =
             mFormat.containsKey(KEY_CROP_RIGHT)
                 && mFormat.containsKey(KEY_CROP_LEFT)
                 && mFormat.containsKey(KEY_CROP_BOTTOM)
                 && mFormat.containsKey(KEY_CROP_TOP);
-        mFormatHasCropValues = Optional.of(hasCropValues);
+        mFormatHasCropValues = Optional.ofNullable(hasCropValues);
```

Comment:
`hasCropValues` won't be null here, so `of` should be good enough?

Reply:
Well, in theory `MediaFormat.containsKey` can throw  because it uses `java.util.Map.containsKey` that can throw, in which case hasCropValues will, albeit unexpectedly, be null.

https://developer.android.com/reference/java/util/Map#containsKey(java.lang.Object)